be/src/http/action/stream_load.cpp

```diff
@@ -234,11 +234,19 @@ Status StreamLoadAction::_on_header(HttpRequest* http_req, StreamLoadContext* ct
     } else {
         ctx->format = parse_format(http_req->header(HTTP_FORMAT_KEY));
         if (ctx->format == TFileFormatType::FORMAT_UNKNOWN) {
-            LOG(WARNING) << "unknown data format." << ctx->brief();
             std::stringstream ss;
             ss << "unknown data format, format=" << http_req->header(HTTP_FORMAT_KEY);
             return Status::InternalError(ss.str());
         }
+
+        size_t max_body_bytes = config::streaming_load_max_batch_size_mb * 1024 * 1024;
+        if (ctx->format == TFileFormatType::FORMAT_JSON) {
+            if (ctx->body_bytes > max_body_bytes) {
+                std::stringstream ss;
+                ss << "body exceed max size of json format: " << ctx->body_bytes << ", limit: " << max_body_bytes;
```

Comment:
the size of this batch exceed the max size of json type data
```suggestion
                ss << "the size of this batch exceed the max size [" << max_body_bytes << "]  of json type data " << " data [ " << ctx->body_bytes << " ] "
```
And I suggest you should truncate the logged `body_bytes` such as just show 1024 byte

Reply:
OK